labs/SParts.md

```diff
@@ -0,0 +1,25 @@
+# Lab Name
+
+SParts Project
+
+# Short Description
+
+The Software Parts (SParts) project delivers a Sawtooth-based Software Parts Ledger that add accountabiity to tracking the open source components used in the construction of a software part whether it be a single library, application, container or an entire operating system runtime. 
+
+# Scope of Lab
+
+This project allows any individual, organization, foundation or supply chain to easily create a distributed ledger to track the open source components used to construct of a collection of software solutions. The initial focus was to track the open source components from which today's manufactured products and devices are constructed. A number of important benefits are obtained by knowing which open source components are used such as: 1) ensuring manufactures are able to identify and secure the distribution rights (licenses) for all open source components; 2) understanding the impact of open source security vulnerabilities; 3) enable identification of cryptography technologies (e.g., FIPS 140-2 certification, export licensing); and 4) enable accurate reporting on all open source parts as a requirement to obtaining functional safety certification for safety critical products (e.g., medical devices, aircraft, autonomous vehicles, elevators, …)  
```

Comment:
"used to construct of a collection" -> "used to construct a collection"
"manufactures" -> "manufacturers"